src/constants.ts

```diff
@@ -9,6 +9,7 @@ export enum SupportedChainId {
 
   ARBITRUM_ONE = 42161,
   ARBITRUM_RINKEBY = 421611,
```

Comment:
Shouldn't rinkeby go away?

Reply:
will probably break some other stuff, might be better to leave it here for now?